authui/src/authflowv2/phoneInput.ts

```diff
@@ -312,6 +318,9 @@ export class PhoneInputController extends Controller {
   connect() {
     void this.initPhoneCode();
     this.phoneInputTarget.addEventListener("blur", this.handleInputBlur);
+    this.phoneInputTarget.parentElement?.classList.remove("hidden");
+    this.countrySelectInputTarget.parentElement?.classList.remove("hidden");
```

Comment:
Maybe target the exact element but not access with `element.parentElement`. Otherwise, it will break when the dom structure is changed